woocommerce/woocommerce-android
# Update login lib with xmlrpc fix

Closes: #3767 


### Description
"Do not merge" label - we need to merge login lib PR first and update the hash.

Integrates changes implemented in [this PR](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/68).

When a site responds with "jetpack connected and active" the app knows that Jetpack connection is working without issues and it can skip the xml-rpc check. This should fix an issue discovered by @designsimply

```
https://wordpress.org/plugins/disable-xml-rpc-api/
When I tried to use the store address > site credentials flow to log in while this plugin is enabled, the error incorrectly tells me that the site at my address is not a valid WordPress site.
```

### Testing instructions

To test in WCAndroid:

1. Install "disable-xml-rpc-api plugin
2. Go into its settings and make sure Jetpack is whitelisted
3. Open "Site address" login flow
4. Enter url of the site
5. Make sure it redirects you to the ".com" login and does NOT display "Not a wordpress site" error
6. 

### Images/gif
